KOA-4589 Define backpack theme (#587)
Skyscanner/backpack-android: README.md

    maven { url 'https://jitpack.io' }
}
```

> To use components update your app theme to inherit from the Backpack theme:

Reviewer: ```suggestion
You also need to inherit your app theme to from the Backpack:
```
